hack/latest-kind-images.sh

```diff
@@ -17,23 +17,20 @@
 
 set -eu -o pipefail
 
-# source this lib for KIND_IMAGE_REPO
-source "devel/lib/lib.sh"
+export KIND_IMAGE_REPO="docker.io/kindest/node"
 
 CRANE=crane
 TAGS=$(mktemp)
 
 trap 'rm -f -- "$TAGS"' EXIT
 
-if ! which $CRANE >/dev/null 2>&1; then
+if ! command -v $CRANE >/dev/null 2>&1; then
```

Comment:
note: I think command -v is more portable (some systems don't have `which` installed)